Library/Formula/bitrise.rb

```diff
@@ -0,0 +1,74 @@
+require "language/go"
+
+class Bitrise < Formula
+  desc "Command-line automation tool"
+  homepage "https://github.com/bitrise-io/bitrise"
+  url "https://github.com/bitrise-io/bitrise/archive/0.9.7.tar.gz"
+  sha256 "cecd0cd112941c6b6f9d6cd334ea6a31aef1123af89e17c3950fbc8e8dbe5de2"
+
+  depends_on "go" => :build
+
+  resource "envman" do
+    url "https://github.com/bitrise-io/envman/archive/0.9.3.tar.gz"
+    sha256 "ff9cb03f978332499bad52f4930f736ded2f8573e8b713b5082f65432e618b8f"
+  end
+
+  resource "stepman" do
+    url "https://github.com/bitrise-io/stepman/archive/0.9.7.tar.gz"
+    sha256 "1eae33616e9c1c952f7c3cfc677fecc0cf0f0781a4aa2e686969e74485284c89"
+  end
+
+  def install
+    go_install_package = lambda do |basepth, go_packagename|
+      mkdir_p "#{basepth}/src/github.com/bitrise-io"
+      ln_s basepth, "#{basepth}/src/github.com/bitrise-io/#{go_packagename}"
+
+      ENV["GOPATH"] = "#{basepth}/Godeps/_workspace:#{basepth}"
+      Language::Go.stage_deps resources, "#{basepth}/src"
+      system "go", "build", "-o", "#{bin}/#{go_packagename}"
+    end
+
+    resource("envman").stage do
+      go_install_package.call(Dir.pwd, "envman")
+    end
+
+    resource("stepman").stage do
+      go_install_package.call(Dir.pwd, "stepman")
+    end
+
+    go_install_package.call(buildpath, "bitrise")
+  end
+
+  test do
+    # simple -version call
+    output = shell_output(bin/"bitrise --version")
+    assert_match "#{version}", output
+
+    # a more complex test - calling a simple bitrise.yml
+    test_bitrise_yml_content = "format_version: 0.9.8
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib
+app:
+  envs: []
+workflows:
+  test_wf:
+    title: Workflow for master branch
+    envs: []
+    steps:
+    - script:
+        title: Test
+        inputs:
+        - content: |-
+            #!/bin/bash
+            set -e
+            pwd
+            printf 'Test - OK' > brew.test.file
+"
+
+    testpath.cd do
+      File.write("bitrise.yml", test_bitrise_yml_content)
```

Comment:
Thanks, done. Quick question: I tried to search for a documentation about what's available in a Formula.rb - which gems/modules in addition to the standard Ruby ones.

Although it seems that `File` is not available by default on the Mountain Lion CI server, but is available on the other two..